src/Classes/ServiceAPI/MyRadio_Scheduler.php

```diff
@@ -143,6 +150,36 @@ public static function getTerm($termid)
 
         throw new MyRadioException('That term could not be found', 400);
     }
+    
+    /**
+     * Returns the current term, if there is one.
+     *
+     * @todo There's currently no caching on this, could be a potential slowdown
+     *
+     * @return Array of a term
+     */
+    
+    public static function getActiveTerm()
```

Comment:
duplicates almost all of getTerms. how much would it break to pull endtime out of getTerms and just use that?

Reply:
Done :)